examples/proxy/server.js

```diff
@@ -130,4 +131,13 @@ app.use(apiPath, (req, res, next) => {
 
 // Start server
 http.createServer(app)
-.listen(port, () => console.log(`Proxy server listening on port ${port}`));
+.listen(port, () => {
+    var url = `http://localhost:${port}`;
+    console.log(`Proxy server listening at ${url}`);
+
+    var cmd = path.join(__dirname, "xdg-open");
+    if (process.platform === "darwin") cmd = "open";
+    else if (process.platform === "win32") cmd = `start ""`;
```

Comment:
Should it not just be `start` instead of `start ""` ?

Reply:
No, windows takes the title as the first parameter.